CHANGELOG.md

```diff
@@ -7,6 +7,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.0] - 2020-04-20
+
+### Added
+
+- [Support for IEEE files](https://gridsuite.github.io/demo/v0.2.0/ieee14.gif)
+- [Creation of a main apps menu](https://gridsuite.github.io/demo/v0.2.0/apps-menu.gif)
+- [Pan in the substation window](https://gridsuite.github.io/demo/v0.2.0/pan-substation-view.gif)
+- [Center the map on a substation on user clicks](https://gridsuite.github.io/demo/v0.2.0/center-map.gif)
+- [All/none actions for the voltage levels on the map](https://gridsuite.github.io/demo/v0.2.0/all-none.gif)
+
+### Changed
+
+- [New fancy study-add button](https://gridsuite.github.io/demo/v0.2.0/add-button.gif)
+- [_undefined_ is no more displayed in the substation list when country info is not available](https://gridsuite.github.io/demo/v0.2.0/country-info.gif)
```

Comment:
unavailable*

Reply:
I think it was "undefined" when there was an empty string because of an unavailable country.